docs/sources/installation/helm/reference.md

```diff
@@ -17,6 +17,11 @@ keywords: []
 
 This is the generated reference for the Loki Helm Chart values.
 
+> **Note:** This reference is for the Loki Helm chart version 3.0 or greater.
+> If you are using the `grafana/loki-stack` Helm chart from the community repo,
```

Comment:
should we add the other "unsupported" charts not replaced by this chart as well, such as `grafana/loki-distributed`?